src/pages/projectSettings/tabs/ViewsAndFiltersTab/transformers.test.ts

```diff
@@ -1,13 +1,16 @@
 import { ProjectHealthView, ProjectSettingsInput } from "gql/generated/types";
 import { data } from "../testData";
+import { ProjectType } from "../utils";
 import { formToGql, gqlToForm } from "./transformers";
 import { ViewsFormState } from "./types";
 
 const { projectBase } = data;
 
 describe("project data", () => {
```

Comment:
i think another test using the repo data showing that the project health view gets correctly omitted would be nice!